From: Andrey Borzenkov <arvidjaar@gmail.com>
Subject: disable rsync to make it possible to use in RPM build

We need to create po/LINGUAS to generate message catalogs. Use
linguas.sh to ensure we always use the same rules as upstream, but
disable rsync.
Index: grub-2.02~rc2/linguas.sh
===================================================================
--- grub-2.02~rc2.orig/linguas.sh
+++ grub-2.02~rc2/linguas.sh
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-rsync -Lrtvz translationproject.org::tp/latest/grub/ po
+#rsync -Lrtvz translationproject.org::tp/latest/grub/ po
 
-autogenerated="en@quot en@hebrew de@hebrew en@cyrillic en@greek en@arabic en@piglatin de_CH"
+autogenerated="en@quot" # en@hebrew de@hebrew en@cyrillic en@greek en@arabic en@piglatin de_CH"
 
 
 for x in $autogenerated; do
